### PR TITLE
Added changes on provisioning stages to allow Jenkins to use deploy playbooks

### PR DESCRIPTION
**Pull request by okynos**
Hello team,

This PR is related with issue #467 we have made changes inside provisioning playbooks, templates and vars to allow Jenkins to use it in an automated way.

Regards.

### CODE REVIEW

File: tests/system/provisioning/roles/aws/tests/route53_records.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
```
Comment (jm404) [inline tests/system/provisioning/roles/aws/tests/route53_records.yml]: This file can be deleted as it was imported from demo-environment but it's not used in this deployment.

File: tests/system/provisioning/vars/credentials.yml [hunk 2]
```diff
@@ -1,16 +1,5 @@
 ### Ansible connection vars
-ssh_username: wazuh
-ssh_password: T3stP4ssw0rd--
-key_pair_name: devops-dev
-
-ssh_key_path_private: ~/.ssh/devops-dev.pem
```
Comment (jm404) [inline tests/system/provisioning/vars/credentials.yml]: This vars are used in `ansible_hosts.j2` template to configure the connection to the instances, e.g:

https://github.com/wazuh/wazuh-qa/blob/f5a28434824ee89be03e91b3e27ac25b9cf400d8/tests/system/provisioning/roles/aws/templates/ansible_hosts.j2#L57

File: tests/system/provisioning/vars/credentials.yml [hunk 3]
```diff
@@ -1,16 +1,5 @@
...
-ssh_key_path_public: ~/.ssh/devops-dev.pub
-
-custom_ssh_key: ""
+ssh_username: jenkins
+key_pair_name: Jenkins
 
 windows_user: Administrator
-windows_password: ""
-
-hosts_classified_destination: "./ec2_demo_hosts_classified"
```
Comment (jm404) [inline tests/system/provisioning/vars/credentials.yml]: These destinations define the path destination for the provisioning playbooks. Please detail motivation for removing them.

File: tests/system/provisioning/vars/aws.yml [hunk 4]
```diff
@@ -52,20 +52,19 @@ kibana_instance_type: t2.large
 ### Stack variables
 
 # Configuration parameters
-stack_name: fim-tests-1
```
Comment (jm404) [inline tests/system/provisioning/vars/aws.yml]: This variable is being used at https://github.com/wazuh/wazuh-qa/blob/07624c6055c6aa7873de92209791b3fa41d86f36/tests/system/provisioning/roles/aws/tasks/launch_template.yml#L31

File: tests/system/provisioning/roles/aws/tests/route53_records.yml (hunk 1 above)
Comment (okynos) [reply tests/system/provisioning/roles/aws/tests/route53_records.yml]: Done e02cb79daa213d847593a9afe11e23edf5481672

File: tests/system/provisioning/vars/credentials.yml (hunk 2 above)
Comment (okynos) [reply tests/system/provisioning/vars/credentials.yml]: Hi!,
You have to set up on the Ansible call due to sensitiveness of such variables
An example command
```
ansible-playbook deploy_aws_stack.yml --private-key /path/to/private.key -u root -e aws_access_key=**** -e aws_secret_key=**** -e aws_region=us-east-1 -e hosts_classified_destination=/any/path/to/hosts_file -e ansible_ssh_config_destination=/any/path/to/hosts_config_file -e stack_name=THE-STACK-NAME -e ssh_key_path_private=/path/to/private.key
```

Regards.

File: tests/system/provisioning/vars/credentials.yml (hunk 3 above)
Comment (okynos) [reply tests/system/provisioning/vars/credentials.yml]: Referred to https://github.com/wazuh/wazuh-qa/pull/509#discussion_r382036642

File: tests/system/provisioning/vars/aws.yml (hunk 4 above)
Comment (okynos) [reply tests/system/provisioning/vars/aws.yml]: Referred to https://github.com/wazuh/wazuh-qa/pull/509#discussion_r382036642

**Comment by alberpilot** [comment]
Merging. Thanks @okynos and @jm404!

**Pull request merged**